aten/src/ATen/native/cuda/SpectralOps.cpp

```diff
@@ -293,7 +309,20 @@ static const Tensor& _exec_fft(Tensor& out, const Tensor& self, IntArrayRef out_
   c10::optional<CuFFTConfig> uncached_plan;
   const CuFFTConfig * config = nullptr;
 
-  if (plan_cache.max_size() > 0) {
+  // Workaround for gh-63152, gh-58724
+  // Bluestein plans in CUDA 11.1 (cufft 10.3) cannot be re-used
+  bool use_caching = true;
+#ifdef CUFFT_VERSION
```

Comment:
Is it possible for types like `CuFFTParams` to be defined but CUFFT_VERSION be undefined?

Reply:
ROCm with hipFFT.